.github/workflows/release-python-playwright.yaml

```diff
@@ -21,8 +21,8 @@ on:
 env:
   RELEASE_TAG: ${{ github.event.inputs.release_tag || github.event.client_payload.release_tag }}
   APIFY_VERSION: ${{ github.event.inputs.apify_version || github.event.client_payload.apify_version }}
-  PLAYWRIGHT_VERSION: ${{ github.event.inputs.playwright_version || github.event.client_payload.playwright_version || '1.33.0' }}
-  LATEST_PYTHON: "3.11"
+  PLAYWRIGHT_VERSION: ${{ github.event.inputs.playwright_version || github.event.client_payload.playwright_version || '1.39.0' }}
+  LATEST_PYTHON: "3.12"
```

Comment:
Let's keep 3.11 as `LATEST_PYTHON` for a while, I bet there are still a lot of libraries broken on 3.12.

Reply:
`LATEST_PYTHON` is bad naming here, it should be called `DEFAULT_PYTHON`, as it denotes the Python version to which the `apify/actor-python` image without any tag will point to. It's called "latest" as it's the `latest` tag of the image.